ui/src/app/components/update-spanner-config-form/update-spanner-config-form.component.ts

```diff
@@ -37,7 +37,15 @@ export class UpdateSpannerConfigFormComponent implements OnInit {
 
     this.fetch.setSpannerConfig(payload).subscribe({
       next: (res: ISpannerConfig) => {
-        this.snack.openSnackBar('Spanner Config updated successfully', 'Close', 5)
+        if (res.IsMetadataDbCreated) {
```

Comment:
Its taking more than 2 mins for creating the metadata db , it wasn't this slow before as far as i remember , any particular change which might have caused this ?

Reply:
It takes around 25-30 second on my local machine. There is only one change which is addition of IsMetadataDbCreated field in the api response. But this won't slow it down as there is this change doesn't cause any heavy computation.

Reply:
Will be happy to discuss more about this in a call.